src/main/java/uk/gov/justice/hmpps/prison/api/resource/OffenderResource.java

```diff
@@ -123,7 +122,7 @@ public InmateDetail getOffender(@Pattern(regexp = "^[A-Z]\\d{4}[A-Z]{2}$", messa
         @ApiResponse(code = 500, message = "Unrecoverable error occurred whilst processing request.", response = ErrorResponse.class)})
     @ApiOperation("Full details about the current state of an offender")
     @GetMapping("/{offenderNo}/prison-timeline")
-    @VerifyOffenderAccess(overrideRoles = {"SYSTEM_USER", "VIEW_PRISONER_DATA"})
+    @PreAuthorize("hasAnyRole('SYSTEM_USER', 'VIEW_PRISONER_DATA')")
```

Comment:
will need to ponder this - these annotations are doing completely different things:
`VerifyOffenderAccess` allows all users to call the endpoint with overrides for those two roles if the user doesn't have privileges
`PreAuthorize` requires that the users must have those two roles in order to access the endpoint.

Need to decide on an endpoint by endpoint basis which is correct I fear

Reply:
I've just moved to @VerifyOffenderAccess to the service layer and pulled @PreAuthorize up to the controller level for consistency